packages/x-components/tests/e2e/cucumber/related-tags/related-tags.spec.ts

```diff
@@ -87,12 +87,12 @@ And(
     if (addToSearchBox) {
       cy.getByDataTest('search-input').should(
         'contain',
-        this.searchedQuery + this.clickedRelatedTag
+        this.searchedQuery + ' ' + this.clickedRelatedTag
```

Comment:
Please use string interpolation ``${this.searchedQuery} ${this.clickedRelatedTag}``